.github/workflows/prerelease.yaml

```diff
@@ -0,0 +1,35 @@
+name: prerelease
+
+on:
+  push:
+    tags:
+      - 'v*.*.*-*'
+
+jobs:
+  prerelease:
+    name: Pre-Release
+    runs-on: ubuntu-latest
+    steps:
+    - name: Set up Go 1.12
+      uses: actions/setup-go@v1
+      with:
+        go-version: 1.12
+      id: go
+
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v1
+
+    - name: Build binaries for all OS/ARCH platforms
+      run: |
+        export PATH=$PATH:$(go env GOPATH)/bin
+        make gox build-all compress=true
+
+    - name: Release
+      uses: softprops/action-gh-release@v1
+      if: startsWith(github.ref, 'refs/tags/')
+      with:
+        files: bin/terraform-docs-v*
+        draft: false
+        prerelease: true
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I think you do not need to specify `GITHUB_TOKEN` while it is automatically provided in `docker run`

https://help.github.com/en/github/automating-your-workflow-with-github-actions/virtual-environments-for-github-actions#creating-and-using-secrets-encrypted-variables